### ISSUE TITLE
Make seed temp/soil moisture/nutrient requirements part of tooltip

### ISSUE THREAD

**Issue by LutaiDunes** [StrangeLoopGames/EcoSuggestions#763]
I feel like there's a lot of data that's impossible to find regarding farming. Right now, you can find the water and temp requirements with the tool, but not only do you have to have a crop planted first - so you have no idea if it'll work out without wasting a seed or testing a wild plant - but also it doesn't indicate anywhere what the optimal nutrients should be. This info exists on the wiki, but not in game. I think the tooltip for the seeds could convey the optimal concentrations of each, and a universal +/- 10% range that the crop will tolerate (with reduced productivity) should be implemented.

**Comment by Guedez** [comment]
I agree that there's no reason for information to exist on the wiki but not the game, but:

It more or less already exists visually in the game (except for the nutrients part)
Since you can just check the map for yield, so you can have a good guess, just not why

Don't seem like a big issue in 8.0, since it seem to be impossible to change temperature and moisture of a tile now, it seems to be static at world creation, so knowing that the issue with the soil is too much moisture will not help.
At least 7.4 techniques don't work

**Comment by LutaiDunes** [comment]
You have a good point, where we could just farm in an area of high yield, but there's and entire specialization and a whole table of recipes for fertilizers. I guess my main gripe is that they should either remove fertilizers and nutrient requirements altogether, or make the information accessible so players can utilize it. 

I'm also sad that water and temperature is entirely dependent on the biome, and no longer changeable. I've sort of been hoping that it's just a feature they're still developing and the ability to influence that will come back.
